test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
+    get users_index_url
+    assert_response :success
+  end
+
+  test 'should get show' do
+    get users_show_url
+    assert_response :success
+  end
+end
```

Comment:
- [ ] Inside `users_spec.rb` file you can write like this as shown in the example code.
```ruby

# spec/requests/users_spec.rb
require 'rails_helper'

 RSpec.describe "Users", type: :request do
   ...
 end
```

and inside `Rspec.describe` you can write `describe` as shown below and inside `describe` you can write `it` block and test the controller.

```ruby
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

- [ ] Spec file include tests that ensures correct view template is rendered. Example : `expect(response).to render_template(:show)`

- [ ] Spec file include tests that ensures the response body includes the correct placeholder text. Example : `expect(response.body).to include("Users#index")`